spyder_terminal/terminalplugin.py

```diff
@@ -119,6 +121,34 @@ def __init__(self, parent):
                                       self, self.create_new_term)
         new_term_shortcut.setContext(Qt.WidgetWithChildrenShortcut)
 
+        self.__wait_server_to_start()
+
+    # ------ Private API ------------------------------------------
+    def __wait_server_to_start(self):
+        try:
+            code = requests.get('http://127.0.0.1:{0}'.format(
+                self.port)).status_code
+        except:
+            code = 500
+
+        if self.server_retries == self.MAX_SERVER_CONTACT_RETRIES:
+            QMessageBox.Critical(self, _('Spyder Terminal Error'),
+                                 _("Terminal server could not be located at "
+                                   '<a href="http://127.0.0.1:{0}">'
+                                   'http://127.0.0.1:{0}</a>,'
+                                   ' please restart Spyder on debugging mode '
+                                   "and open an issue with the contents of "
+                                   "<tt>{1}</tt> and <tt>{2}</tt> "
+                                   "files.".format(self.port,
```

Comment:
Let's improve this a bit

    files at {3}.

where `{3}` is

     https://github.com/spyder-ide/spyder-terminal/issues